staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1232,7 +1249,7 @@ func (r *Request) DoRaw(ctx context.Context) ([]byte, error) {
 }
 
 // transformResponse converts an API response into a structured API object
-func (r *Request) transformResponse(resp *http.Response, req *http.Request) Result {
+func (r *Request) transformResponse(ctx context.Context, resp *http.Response, req *http.Request) Result {
```

Comment:
Since this method already has a ctx parameter, why aren’t lines [1267](https://github.com/kubernetes/kubernetes/pull/126999/files#diff-e6b16db60c379a7e5fdb8f46054d9a87ea511dba3f94d261b5f61a506fcb5be2R1267) and [1273](https://github.com/kubernetes/kubernetes/pull/126999/files#diff-e6b16db60c379a7e5fdb8f46054d9a87ea511dba3f94d261b5f61a506fcb5be2R1267) formatted as `logger.V(2).Info() `and logger.Error() respectively?  Other updated methods also are similar.

Reply:
I was focusing on just the problem with request and response. But you are right, it probably makes more sense to convert all of client-go/rest.

Reply:
I converted also the rest of client-go/rest (see https://github.com/kubernetes/kubernetes/compare/master...pohly:kubernetes:log-client-go-rest?expand=1). There were a few other places with non-trivial changes, so I'm leaning towards merging the changes commit-by-commit in separate PRs.

cc @sttts

Reply:
> I was focusing on just the problem with request and response. But you are right, it probably makes more sense to convert all of client-go/rest.

That makes sense.